paasta_tools/cli/cmds/secret.py

```diff
@@ -444,7 +444,7 @@ def paasta_secret(args):
         secret_provider = _get_secret_provider_for_service(
             service,
             cluster_names=args.clusters,
-            soa_dir=args.yelpsoa_config_root,
+            soa_dir="./",
```

Comment:
thoughts on having this be something like:
```suggestion
            # this will only be invoked on a devbox
            # and only in a context where we certainly
            # want to use the working directory rather
            # than whatever the actual soa_dir path is
            # configured as
            soa_dir=os.getcwd(),
```